samples/client/petstore/python-experimental/test/test_additional_properties_any_type.py

```diff
@@ -5,7 +5,7 @@
 
     This spec is mainly for testing Petstore server and contains fake endpoints, models. Please do not use this for any other purpose. Special characters: \" \\  # noqa: E501
 
```

Comment:
Please revert all the python-experimental/tests file back to the way they originally were
That way it is easier to review this PR as only applying to the python generator

Reply:
Done

Reply:
Can you resolve this conversation?